charts/osm/templates/crds-upgrade-hook-rbac.yaml

```diff
@@ -0,0 +1,103 @@
+{{- if and (not (.Capabilities.APIVersions.Has "security.openshift.io/v1")) .Values.OpenServiceMesh.pspEnabled }}
```

Comment:
Lets not create a separate file for the rbac needed by the hook. all the required resources for the hook can be in the same file